index.js

```diff
@@ -177,15 +177,21 @@ const earlyExpiration = argv.enableEarlyExpiration ? argv.enableEarlyExpiration
 
   // Simulate transaction to test before sending to the network.
   console.log("Simulating Deployment...");
-  const address = await lspCreator.methods.createLongShortPair(lspParams).call(transactionOptions);
-  console.log("Simulation successful. Expected Address:", address);
+  await lspCreator.methods.createLongShortPair(lspParams).call(transactionOptions);
```

Comment:
The address returned by the simulation is not the same as the real address when we do the actual deployment some lines below